fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml

```diff
@@ -416,6 +416,28 @@ paths:
         "401":
           $ref: "#/components/responses/401_Unauthorized"
 
+  /v1/consent/{userid}/{password}:
+    get:
+      summary: ask for existing consent of user
```

Comment:
Add note that it is a temporary endpoint for debug

Reply:
Its written in description, that is depends on spring property.